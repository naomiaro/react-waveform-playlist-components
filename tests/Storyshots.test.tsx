import initStoryshots from '@storybook/addon-storyshots';
import 'jest-styled-components';
import { mount } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

initStoryshots({
  renderer: mount,
  //@ts-ignore
  snapshotSerializers: [createSerializer()],
});
