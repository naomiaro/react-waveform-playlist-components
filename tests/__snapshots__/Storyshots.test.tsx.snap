// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Channel Default 1`] = `null`;

exports[`Storyshots Track Controls Default 1`] = `null`;

exports[`Storyshots WebAudio Default 1`] = `null`;

exports[`Storyshots WebAudio Multi Channel 1`] = `null`;

exports[`Storyshots WebAudio With Knobs 1`] = `null`;
